File: server/trunk/server/src/config/guild_config.erl
%%%-------------------------------------------------------------------
%%% @author zhengsiying
%%% @doc
%%%		自动生成文件，不要手动修改
%%% @end
%%% Created : 2016/10/12
%%%-------------------------------------------------------------------

-module(guild_config).

-include("common.hrl").
-include("config.hrl").

-compile([export_all]).

get_list_conf() ->
	[ guild_config:get(X) || X <- get_list() ].

get_list() ->
	[1, 2, 3, 4, 5, 6, 7, 8, 9, 10].

get(1) ->
	#guild_conf{
		key = 1,
		exp = 9000,
		member_limit = 40
	};

get(2) ->
	#guild_conf{
		key = 2,
		exp = 20000,
		member_limit = 45
	};

get(3) ->
	#guild_conf{
		key = 3,
		exp = 60000,
		member_limit = 50
	};

get(4) ->
	#guild_conf{
		key = 4,
		exp = 130000,
		member_limit = 55
	};

get(5) ->
	#guild_conf{
		key = 5,
		exp = 260000,
		member_limit = 60
	};

get(6) ->
	#guild_conf{
		key = 6,
		exp = 520000,
		member_limit = 65
	};

get(7) ->
	#guild_conf{
		key = 7,
		exp = 1040000,
		member_limit = 70
	};

get(8) ->
	#guild_conf{
		key = 8,
		exp = 2080000,
		member_limit = 75
	};

get(9) ->
	#guild_conf{
		key = 9,
		exp = 4160000,
		member_limit = 80
	};

get(10) ->
	#guild_conf{
		key = 10,
		exp = 9999999999,
		member_limit = 85
	};

get(_Key) ->
	?ERR("undefined key from guild_config ~p", [_Key]).

File: client/trunk/src_lua/app/conf/guildConfig.lua
-- 自动生成，请勿修改 
-- 时间：2017/01/15
-- 21102585@qq.com

local guildConfig = class("guildConfig")
function guildConfig:ctor()
	self.fields = {"key", "exp", "member_limit"}
	self.datas = {
		[1] = {1, 9000, 40},
		[2] = {2, 20000, 45},
		[3] = {3, 60000, 50},
		[4] = {4, 130000, 55},
		[5] = {5, 260000, 60},
		[6] = {6, 520000, 65},
		[7] = {7, 1040000, 70},
		[8] = {8, 2080000, 75},
		[9] = {9, 4160000, 80},
		[10] = {10, 9999999999, 85},

	}
end
return guildConfig
